docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,189 @@ Authorization
+|                                     | ListUserRights             | authenticated users can list their own rights          |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+| VersionService                      | All                        | public                                                 |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+
+
+.. _access-token-formats:
+
+Access token formats
+********************
+
+Applications should treat access tokens as opaque blobs.
+However as an application developer it can be helpful to understand the format of access tokens to debug problems.
+
+All Daml ledgers represent access tokens as `JSON Web Tokens (JWTs) <https://datatracker.ietf.org/doc/html/rfc7519>`_,
+and there are two formats of the JSON payload in use by Daml ledgers.
+
+
+Custom Daml claims access tokens
+================================
+
+This format represents the :ref:`rights <authorization-claims>` granted by the access token as custom claims in the JWT's payload, like so:
+
+
+.. code-block:: json
+
+   {
+      "https://daml.com/ledger-api": {
+        "ledgerId": null,
+        "participantId": "123e4567-e89b-12d3-a456-426614174000",
+        "applicationId": null,
+        "admin": true,
+        "actAs": ["Alice"],
+        "readAs": ["Bob"]
+      },
+      "exp": 1300819380
+   }
+
+where all of the fields are optional, and if present,
+
+- ``ledgerId`` and ``participantId`` restrict the validity of the token to the given ledger or participant node
+- ``applicationId`` requires requests with this token to use that application id or not set an application id at all, which should be used to distinguish requests from different applications
+- ``exp`` is the standard JWT expiration date (in seconds since EPOCH)
+- ``actAs``, ``readAs`` and (participant) ``admin`` encode the rights granted by this access token
+
+The ``public`` right is implicitly granted to any bearing a valid JWT issued by a trusted issuer (even without being an admin or being able to act or read on behalf of any party).
+
+.. note:: All Daml ledgers also support a deprecated legacy format of custom Daml claims
+   access tokens whose format is equal to the above expect for the custom claims
+   to be present at the same level as ``exp`` in the token above,
+   instead of being nested below ``"https://daml.com/ledger-api"``.
+
+
+User access tokens
+==================
+
+Daml ledger that support participant user management also accept user access tokens.
+They are useful for scenarios where an application's rights change dynamically over the application's lifetime.
+
+User access tokens do not encode rights directly like custom Daml claims tokens.
+Instead, user access tokens encode the participant user on behalf of which the application is issuing a request.
```

Comment:
"Instead, these tokens encode the participant user on whose behalf the request is issued."